src/ttsnake.c

```diff
@@ -356,6 +357,29 @@ void showscene (scene_t* scene, int number, int menu)
  * energy blocks until it finds the inactive block. It then replaces it, and ends.*/
 void more_snacks(){
    /* Generate energy blocks away from the borders and the snake */
+
+ 	int i = 0, j = 0, isValid;
+	/*We generate a new block and check it against every piece of the snake.
+	 if the block is on top of the snake, then we generate a new position
+	 for it. Otherwise, we go to the next block and do it again.*/
+    	do
+  	{
+  	  isValid = 1;
+	    energy_block[i].x = (rand() % (NCOLS - 2)) + 1;
+  	  energy_block[i].y = (rand() % (NROWS - 2)) + 1;
+	  for(j = 0; j < snake.length;j++){
+		  if(energy_block[i].x == snake.positions[j].x && energy_block[i].y == snake.positions[j].y){
+			  isValid = 0;
+			  break;
+		  }
+	  }
+	  if(isValid == 1){
+		  i++;
+	  }
+
+  	}while(i < MAX_ENERGY_BLOCKS);
+
+=======
```

Comment:
small error here, but I'll try to fix it myself

Reply:
> I am unable to compile. Did you manage to do so?
> 
> There are variables being redefined, and two blocks of code that do the same thing.
> 
> I think you might have messed things up when resolving conflicts of a merge.
> 
> The error:
> 
> ```
> ttsnake.c: In function ‘more_snacks’:
> ttsnake.c:382:7: error: redefinition of ‘i’
>    int i = 0, j, isValid = 0;
>        ^
> ttsnake.c:361:7: note: previous definition of ‘i’ was here
>    int i = 0, j = 0, isValid;
>        ^
> ttsnake.c:382:14: error: redeclaration of ‘j’ with no linkage
>    int i = 0, j, isValid = 0;
>               ^
> ttsnake.c:361:14: note: previous definition of ‘j’ was here
>    int i = 0, j = 0, isValid;
>               ^
> ttsnake.c:382:17: error: redeclaration of ‘isValid’ with no linkage
>    int i = 0, j, isValid = 0;
>                  ^~~~~~~
> ttsnake.c:361:21: note: previous declaration of ‘isValid’ was here
>    int i = 0, j = 0, isValid;
>                      ^~~~~~~
> ```

Turns out when I tried to resolve conflicts I didn't remove the previous version of more_snacks() so variables where declared twice, I removed the previous iteration and matched it to the develop branch, will try testing it now.